### PR TITLE
Added Raspberry PI book list and two other books.

### PR DESCRIPTION
**Pull request by GintasS**
## What does this PR do?
Add Resources

### Description 
Added Raspberry PI meta list, added books made by Al Sweigart on Scratch and Minecraft programming.

### Why is this valuable (or not)?
There are lot of books here, from Raspberry PI, Scratch and Minecraft.

### How do we know it's really free?
Raspberry PI magazine books are free obviously, and books by Al Sweigart are available on Al's websites.

### For book lists, is it a book? For course lists, is it a course? etc.
There is one book list and two normal books.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -2439,6 +2440,7 @@ Kerridge (PDF) (email address *requested*, not required)
 
 ### Raspberry Pi
 
+* [Raspberry PI Books](https://magpi.raspberrypi.org/books) - The official Raspberry Pi magazine (PDF) (meta-list)
```
Comment (eshellman) [inline free-programming-books.md:2443]: we don't include lists of books in our lists of books.

**Review by eshellman — CHANGES_REQUESTED** [review]
Like the Al Sweigart books!

File: free-programming-books.md (hunk 1 above)
Comment (GintasS) [reply free-programming-books.md:2443]: So should we not include the Raspberry Pi books at all mentioned on the issue, or do I need to add them on one by one basis?

Comment (GintasS) [reply free-programming-books.md]: @eshellman I removed the list, let me know if I should add the books one by one.

**Comment by GintasS** [comment]
@eshellman I removed the list, let me know if I should add the books one by one.

**Pull request merged**